troposphere/static/js/components/providers/Instances.react.js

```diff
@@ -34,7 +34,10 @@ define(function (require) {
 
     render: function () {
       var provider = this.props.provider,
-        instances = stores.InstanceStore.findWhere({'provider.id': provider.id}),
+        instances = stores.InstanceStore.findWhere({
+            'provider.id': provider.id,
+            'status': 'active'
+        }),
```

Comment:
What network request(s) go out when this call is made? I don't believe that the API will return filters based on status (So this might be misleading?)